### PR TITLE
Updated dbt utils package version

### PR DESCRIPTION
**Pull request by andrewbeyerly**
Validated by running `dbt compile`. 

Doing this so we can [upgrade](https://github.com/babylist/dbt_models/pull/2603) the main dbt repo

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -37,4 +38,6 @@ vars:
 
   # BigQuery only: partition granularity for partition_by clause
   # options can be found here: https://docs.getdbt.com/reference/resource-configs/bigquery-configs#partition-clause
-  segment_bigquery_partition_granularity: "day"
\ No newline at end of file
+  segment_bigquery_partition_granularity: "day"
+  surrogate_key_treat_nulls_as_empty_strings: true
+
```
Comment (arthurnw) [inline dbt_project.yml:43]:
```suggestion

```

File: models/sessionization/segment_web_page_views__sessionized.sql [hunk 2]
```diff
@@ -152,7 +152,7 @@ session_ids as (
         session_starts.session_start_tstamp,
         --if an event has previously been sessionized, keep the existing session ID
         {% if is_incremental() %}sessionized.session_id{% else %}null::string{% endif %} as existing_session_id,
-        {{dbt_utils.surrogate_key(['session_starts.anonymous_id', 'session_starts.session_start_tstamp', 'session_starts.session_number'])}} as session_id
+        {{dbt_utils.generate_surrogate_key(['session_starts.anonymous_id', 'session_starts.session_start_tstamp', 'session_starts.session_number'])}} as session_id
```
Comment (arthurnw) [inline models/sessionization/segment_web_page_views__sessionized.sql:155]: Why is this still `dbt_utils`?

File: packages.yml [hunk 3]
```diff
@@ -1,3 +1,3 @@
 packages:
  - package: dbt-labs/dbt_utils
-   version: [">=0.8.0", "<0.9.0"]
+   version: ">=1.3.0"
```
Comment (arthurnw) [inline packages.yml:3]: Should we maybe pin this to `==`? Our stuff usually runs `dbt deps` at the beginning of the job, so we could potentially get an inadvertent upgrade in a prod job.

File: dbt_project.yml [hunk 4]
```diff
@@ -2,6 +2,7 @@ name: 'segment'
 version: '0.7.0'
 require-dbt-version: [">=1.0.0", "<2.0.0"]
 config-version: 2
+profile: babylist_analytics_snowflake
```
Comment (arthurnw) [inline dbt_project.yml:5]: Do we need this line? I'm not sure what would happen here; I guess having this just makes it runnable locally.

File: dbt_project.yml [hunk 5]
```diff
@@ -37,4 +38,5 @@ vars:
 
   # BigQuery only: partition granularity for partition_by clause
   # options can be found here: https://docs.getdbt.com/reference/resource-configs/bigquery-configs#partition-clause
-  segment_bigquery_partition_granularity: "day"
\ No newline at end of file
+  segment_bigquery_partition_granularity: "day"
+  surrogate_key_treat_nulls_as_empty_strings: true
```
Comment (arthurnw) [inline dbt_project.yml:42]:
```suggestion
  surrogate_key_treat_nulls_as_empty_strings: true

```

File: dbt_project.yml (hunk 4 above)
Comment (andrewbeyerly) [reply dbt_project.yml:5]: Yeah I had put this in here so I could compile the project locally and work through any errors

File: models/sessionization/segment_web_page_views__sessionized.sql (hunk 2 above)
Comment (andrewbeyerly) [reply models/sessionization/segment_web_page_views__sessionized.sql:155]: I think I misspoke on our zoom earlier, only cross-db macros have changed namespaces.  Others (like the star macro) will remain `dbt_utils`

**Review by arthurnw — APPROVED** [review]

**Pull request merged**